HUD/HUD.gd

```diff
@@ -1,6 +1,12 @@
 extends CanvasLayer
 
-signal stream_ready(stream)
+var MusicAPI
+
+func _ready(): MusicAPI = get_tree().get_nodes_in_group("MusicAPI")[0]
```

Comment:
This feels kinda convoluted, I'll have this changed to an autoload :)

Reply:
Noted, making changes asap!

Reply:
Fixed autoload!